### PR TITLE
fix: cache file is place wrong path, download to other cache place, if ok then mv to cache file

### PR DESCRIPTION
**Pull request by Decodetalkers**
Issue: https://github.com/linuxdeepin/developer-center/issues/6622 Log:

### CODE REVIEW

File: dcc-deepinid-plugin/pages/downloadurl.cpp [hunk 1]
```diff
@@ -5,27 +5,35 @@
...
-#include <QTimer>
+#include <QNetworkRequest>
 #include <QPixmap>
-#include <QDebug>
+#include <QStandardPaths>
+#include <QTimer>
 
 DownloadUrl::DownloadUrl(QObject *parent)
     : QObject(parent)
+    , m_cachePath(QStandardPaths::writableLocation(QStandardPaths::CacheLocation)
```
Comment (kegechen) [inline dcc-deepinid-plugin/pages/downloadurl.cpp]: QTemporaryFile ?

File: dcc-deepinid-plugin/pages/downloadurl.cpp [hunk 2]
```diff
@@ -5,39 +5,49 @@
...
 
 DownloadUrl::~DownloadUrl()
 {
-    if (m_manager != nullptr)
+    if (m_manager != nullptr) {
         m_manager->deleteLater();
+    }
+    if (m_file != nullptr) {
+        m_file->deleteLater();
+    }
```
Comment (kegechen) [inline dcc-deepinid-plugin/pages/downloadurl.cpp:36]: 虽然但是，qt风格一行的判断貌似可以不需要{}

**Review by kegechen — COMMENTED** [review]
+1

**Review by 18202781743 — APPROVED** [review]

**Pull request merged**